spirit/topic/templates/spirit/topic/_render_list.html

```diff
@@ -18,17 +18,25 @@
                 </div>
                 <div class="row-info">
                     <div>
+
+
                         {% if show_subcategory %}
-                            <a href="{{ t.category.get_absolute_url }}">{{ t.category.title }}</a>
+                            <a {% if t.category.colour != '' %}style="color: {{ t.category.colour }};"{% endif %} href="{{ t.category.get_absolute_url }}">
+                                {{ t.category.title }}
+                            </a>
                         {% else %}
                             {% if t.category.parent %}
-                                <a href="{{ t.category.parent.get_absolute_url }}">{{ t.category.parent.title }}</a>
+                                <a {% if t.category.colour != '' %}style="color: {{ t.category.colour }};"{% endif %} href="{{ t.category.parent.get_absolute_url }}">
+                                    {{ t.category.parent.title }}
+                                </a>
                             {% else %}
-                                <a href="{{ t.category.get_absolute_url }}">{{ t.category.title }}</a>
+                                <a {% if t.category.colour != '' %}style="color: {{ t.category.colour }};"{% endif %} href="{{ t.category.get_absolute_url }}">
+                                    {{ t.category.title }}
+                                </a>
                             {% endif %}
                         {% endif %}
-                    </div><!--
-                 --><div><i class="fa fa-comment{% if t.has_new_comments %} js-comment-is-new{% endif %}"></i> {{ t.comment_count }}</div><!--
+                    </div>
+                    <div><i class="fa fa-comment{% if t.has_new_comments %} js-comment-is-new{% endif %}"></i> {{ t.comment_count }}</div><!--
```

Comment:
The comment thing `<!-- ... -->` is on porpuse, it removes the white space between inline-elements.